pkg/datapath/iptables/cell.go

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package iptables
+
+import (
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/cidr"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/option"
+	"github.com/cilium/cilium/pkg/time"
+)
+
+const (
```

Comment:
i'd prefer having these const values inlined. It's still possible to keep the comments above the corresponding flag definitions in `Flags(...)`

Reply:
Yep likewise. I would move the doc comments on to the struct fields.